assets/extensions/examples/online-extensions.js

```diff
@@ -46,7 +46,6 @@ angular
 
       $.getJSON("https://m0sg3q4t415n.statuspage.io/api/v2/summary.json", function (data) {
         var n = (data.incidents || [ ]).length;
-        n = 32;
```

Comment:
Self-deprecating shame for letting test code (`n=32`) slide in last time :disappointed: